frontend/src/features/shared/components/storybook-preview.scss

```
@@ -0,0 +1,14 @@
+@import '../../../static/css/config/index.scss';
+@import '../../../static/css/includes/typography/_text_styles.scss';
```

Stylelint (17.9.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)